CefSharp.Core/Internals/CefDragDataWrapper.h

```diff
@@ -41,6 +41,24 @@ namespace CefSharp
             virtual property bool IsFragment;
             virtual property bool IsLink;
 
+            ///
+            // Create a new CefDragData object.
+            ///
+            /*--cef()--*/
+            static CefDragDataWrapper^ Create(){
+                CefRefPtr<CefDragData> cefDragData = CefDragData::Create();
+                return gcnew CefDragDataWrapper(cefDragData);
+            }
+
+
+            virtual property CefRefPtr<CefDragData>* _internalDragData
```

Comment:
Not really a fan of naming a property starting with a `_`, calling it `InternalDragData` should be fine.